EventFilter/GEMRawToDigi/plugins/GEMDigiToRawModule.cc

```diff
@@ -130,7 +130,6 @@ void GEMDigiToRawModule::produce(edm::StreamID iID, edm::Event& iEvent, edm::Eve
     for (uint8_t amcNum = 0; amcNum < GEMeMap::maxAMCs_; ++amcNum) {
       uint32_t amcSize = 0;
       std::unique_ptr<AMCdata> amcData = std::make_unique<AMCdata>();
-      amcData->setAMCheader1(amcSize, BX_id, LV1_id, amcNum);
```

Comment:
is this header assignment part really not needed?

Reply:
yes, it is set later on anyways.
I was mainly using it for the 16bit to 12bit conversion of BX_id

Reply:
IIUC, it will be set only if `!gebData->vFATs()->empty()`.
I suppose a real AMC without any VFATs will still send a header.
Perhaps the point for MC is that it doesn't matter.

Reply:
Yes, for MC it doesn't matter.
I can remove the if !gebData->vFATs()->empty() if you prefer it.

Reply:
naively, a better representation of reality would make more sense. E.g. with the headers/trailers properly set the unpacker would be more capable of doing integrity checks without some kind of `if(realData)`. But perhaps it's an academic topic and it doesn't really matter.

I don't have a strong preference.

Reply:
I've changed to always producing AMCs.